src/menu/example/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Menu Example</title>
+</head>
+<body>
+	<script src="../../../../node_modules/@dojo/loader/loader.min.js"></script>
+	<link rel="stylesheet" href="../../common/styles/widgets.css">
+	<link rel="stylesheet" href="./styles/app.css">
```

Comment:
Sounds good. I knew that was being separated out eventually, so I just did that here. I'll move it back into the main `menu.css`, perhaps with notes disambiguating any styles that could go either way.